.github/workflows/smoke-tests-manual.yml

```diff
@@ -11,7 +11,7 @@ on:
       os:
         required: false
         type: string
-        default: '["macos-latest-xl", "ubuntu-latest", "windows-latest"]'
+        default: '["macos-latest-xl", "ubuntu-latest", "amplify-cli_windows-latest_8-core"]'
```

Comment:
I have attempted to make this list a bit smoother i.e. `macos,ubuntu,windows`. However, it makes actions quite complicated, so I parked that effort.
End of the day, when we run this manually, this parameter is populated with this default, all operator has to do is to remove values.

Reply:
Where is the `amplify-cli_windows-latest_8-core` from?

Reply:
https://t.corp.amazon.com/P98348294/communication

Reply:
Also a visual.
<img width="1154" alt="image" src="https://github.com/aws-amplify/amplify-cli/assets/5849952/0f709fd9-011a-46c4-b801-27e0e7b824d7">